api/handlers.go

```diff
@@ -557,6 +559,125 @@ func (si *ServerImplementation) LookupApplicationByID(ctx echo.Context, applicat
 	})
 }
 
+// LookupApplicationBoxByIDAndName returns the value of an application's box
+// (GET /v2/applications/{application-id}/box)
+func (si *ServerImplementation) LookupApplicationBoxByIDAndName(ctx echo.Context, applicationID uint64, params generated.LookupApplicationBoxByIDAndNameParams) error {
+	if err := si.verifyHandler("LookupApplicationBoxByIDAndName", ctx); err != nil {
```

Comment:
@AlgoStephenAkiki could you confirm that `verifyHandler` is being used correctly? I don't have any reason to suspect it isn't, just would like you to confirm.

Reply:
I'm using it in two places.

Reply:
Looks good.  As long as the string matches the function name exactly you should be good.